packages/crypto/src/lib/crypto.ts

```diff
@@ -270,14 +270,20 @@ export const combineEcdsaShares = (
         log('response from combine_signature', res);
         sig = JSON.parse(res);
         break;
+      case SIGTYPE.FrostEd25519Sha512v1:
+        res = ecdsaSdk.combine_signature(validShares, 4);
```

Comment:
Is there an update to the `combiner` that follows this new `FrostEd25519Sha512v1` if so can you link that as well in this pr?

Reply:
Right! Here: https://github.com/LIT-Protocol/lit-ecdsa-wasm-combine/pull/5 (will add to the description as well)